src/vs/base/parts/quickopen/browser/quickOpenWidget.ts

```diff
@@ -793,6 +810,10 @@ export class QuickOpenWidget extends Disposable implements IModelProvider {
 		this.builder.hide();
 		this.builder.domBlur();
 
+		// In case of quick search store entered value
+		if (!this.withPrefix) {
```

Comment:
Note that one can start with a prefix, but then delete it and the other way around.

Reply:
I I've considered that now.